micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login/login.js

```
@@ -131,9 +136,14 @@ const Login = ({ config: propsConfig, t, isDisabled, loginOTPBased }) => {
           setTimeout(closeToast, 5000);
         },
         onSuccess: async (data) => {
-          history.push({
-            pathname: `/${window?.contextPath}/employee/user/login/otp`,
-            state: { email: inputEmail, tenant: Digit.ULBService.getStateId() },
+          console.log(`*** LOG DATA***`, data);
+          console.log(`*** LOG DATA***`, window?.contextPath);
+          let redirectPath = `/${window?.contextPath}/${tenantId}/employee/user/login/otp`;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'redirectPath' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)